src/api/volume.ts

```diff
@@ -337,59 +336,4 @@ const getVolumeDaily = async (assetId?: string) => {
   return data
 }
 
-export function useAllStableswapTrades() {
-  const indexerUrl = useIndexerUrl()
-
-  return useQuery(
-    QUERY_KEYS.allStableswapTrades,
-    getAllStableswapTrades(indexerUrl),
-    {
-      select: useCallback((data: { events: TradeType[] }) => {
-        return data.events.reduce<Record<string, TradeType[]>>((acc, event) => {
-          acc[event.args.assetIn]
-            ? acc[event.args.assetIn].push(event)
-            : (acc[event.args.assetIn] = [event])
-
-          return acc
-        }, {})
-      }, []),
-      refetchInterval: 60000,
-    },
-  )
-}
-
-export const getAllStableswapTrades = (indexerUrl: string) => async () => {
-  const after = addDays(new Date(), -1).toISOString()
-
-  // This is being typed manually, as GraphQL schema does not
-  // describe the event arguments at all
-  return {
-    ...(await request<{
-      events: Array<TradeType>
-    }>(
-      indexerUrl,
-      gql`
-        query TradeVolume($after: DateTime!) {
-          events(
-            where: {
-              name_eq: "Stableswap.SellExecuted"
-              block: { timestamp_gte: $after }
-              OR: {
-                name_eq: "Stableswap.BuyExecuted"
-                block: { timestamp_gte: $after }
-              }
-            }
-          ) {
-            id
-            name
-            args
-            block {
-              timestamp
-            }
-          }
-        }
-      `,
-      { after },
-    )),
-  }
-}
+export const envStableCoinId = import.meta.env.VITE_STABLECOIN_ASSET_ID
```

Comment:
do we still need to take stable coin ID from env variables?

Reply:
yes, in case when stablecoin id is not loaded from rpc yet